.github/workflows/test.yml

```diff
@@ -90,3 +90,18 @@ jobs:
       - uses: actions/checkout@v4
       - uses: streetsidesoftware/cspell-action@v6
 
+  check_rust_backward_compatibility_1_81:
```

Comment:
it would make more sense to me if it were made a one more element in `matrix.toolchain` array on line 68 instead of a separate `check_rust_backward_compatibility_1_81` job 

also the line 84 `cargo check --verbose` could be changed to => `cargo check --examples --all-features` (`--verbose` is annoying in ci and examples aren't checked at all in ci)
and line 86 `cargo test --verbose` could be changed to `cargo test`

Reply:
also doing the following in this case would be required: 
```yaml
        env:
          # info: (overridden by './rust-toolchain.toml')
          RUSTUP_TOOLCHAIN: ${{ matrix.toolchain }}
```

https://github.com/near/cargo-near/blob/main/.github/workflows/test.yml#L39-L41

Reply:
also a `rust-version = 1.81` line could be added into manifest here https://github.com/near/near-api-rs/blob/readme-update/Cargo.toml#L4

Reply:
`cargo-near` has gone great lengths to keep this only in manifest and to have it auto-updated in ci https://github.com/near/cargo-near/blob/main/.github/workflows/test.yml#L22-L25

Reply:
The thing is that 1.81 only works with no features enabled. So, the matrix doesn't really make sense. As well, as putting it into Cargo.toml. As it would state this is fully supported by the crate.